src/platform/site-wide/index.js

```diff
@@ -22,6 +22,10 @@ if (!brandConsolidation.isEnabled()) {
   require('./usa-banner-toggle');
 }
 
+if (brandConsolidation.isEnabled()) {
+  import(/* webpackChunkName: "uswds" */ 'uswds');
```

Comment:
@rianfowler Is this going to break in Teamsite? Seems like it could for subdomain pages.

Reply:
I don't know if this will break team site.  I'll test it and see what happens

Reply:
This seems fine. I built this branch and proxied the vendor and proxy-rewrite entry files on `http://www.cem.va.gov` and I was able to get the header to show up. 

Also verified that the header still works on preview.va.gov/health with those two scripts.